interface/main/main_title.php

```diff
@@ -87,6 +93,10 @@ function showhideMenu() {
             <a href='' class="css_button_small" style="margin:0px;vertical-align:top;display:none;" id='clear_active' onClick="javascript:parent.left_nav.clearactive();return false;">
             <span><?php echo htmlspecialchars( xl('CLEAR ACTIVE PATIENT'), ENT_QUOTES); ?></span></a>
     </td>
+    <td style="vertical-align:text-bottom;">
```

Comment:
yes,  and the setPatient function in left_nav.php:782 (at the end) hides this element. therefore both main_title elements will never show up together